apple/testing/default_runner/ios_test_runner.template.sh

```diff
@@ -102,12 +117,23 @@ if [[ -n "${LAUNCH_OPTIONS_JSON_STR}" ]]; then
   runner_flags+=("--launch_options_json_path=${LAUNCH_OPTIONS_JSON_PATH}")
 fi
 
+target_flags=()
+if [[ -n "$simulator_id" ]]; then
+  target_flags=(
+    "test"
+    "--id=$simulator_id"
```

Comment:
Please also add --platform=ios_simulator to the runner. It can help xctestrunner skip the platform check.

Reply:
Done!